lib/helpers/fields/statistics-fields.js

```diff
@@ -0,0 +1,102 @@
+const _ = require('lodash')
+const { isDate } = require('./data-type-utils')
+const {
+  StatisticField,
+  StatisticDateField,
+  FieldFromFieldDefinition,
+  FieldFromKeyValue
+} = require('./field-classes')
+
+class StatisticsFields {
+  static normalizeOptions (inputOptions) {
+    const {
+      statistics,
+      metadata: {
+        fields
+      } = {},
+      groupByFieldsForStatistics = [],
+      attributeSample,
+      ...options
+    } = inputOptions
+
+    return {
+      statisticsSample: Array.isArray(statistics) ? statistics[0] : statistics,
+      fieldDefinitions: options.fieldDefinitions || options.fields || fields,
+      groupByFieldsForStatistics: Array.isArray(groupByFieldsForStatistics) ? groupByFieldsForStatistics : groupByFieldsForStatistics
+        .replace(/\s*,\s*/g, ',')
+        .replace(/^\s/, '')
+        .replace(/\s*$/, '')
+        .split(','),
+      ...options
+    }
+  }
+
+  static create (inputOptions) {
```

Comment:
Should `inputOptions` need a default `{}`?

Reply:
good idea

Reply:
done